DNN Platform/Library/Entities/Users/UserController.cs

```diff
@@ -2028,7 +2083,11 @@ internal static void UpdateUser(int portalId, UserInfo user, bool loggedAction,
                 DataCache.ClearUserCache(portalId, user.Username);
             }
 
-            if (!user.Membership.Approving) return;
+            if (!user.Membership.Approving)

```

Comment:
Can this format change back to the inlined version for consistency with the rest of the platform

Reply:
ctrl-K ctrl-D does this for me in VS. I'll change it back.

Reply:
Done